e2e-tests/tests/rearrange-elements.spec.ts

```diff
@@ -8,22 +8,31 @@ const expectedPage = fs.readFileSync(
 );
 
 studioTest("can rearrange elements in tree", async ({ page, studioPage }) => {
-  const banner = page.getByText("Banner");
-  const div = page.getByRole("list").filter({ hasText: "div" });
-  const divBox = await div.boundingBox();
+  await studioPage.switchPage("LocationPage");
+  await studioPage.addElement("Footer", "Components", false);
+  await expect(page).toHaveScreenshot();
+  const footer = page.getByText("Footer");
+  const businessInfo = page
+    .getByRole("list")
+    .filter({ hasText: "BusinessInfo" });
+  const businessInfoBox = await businessInfo.boundingBox();
 
-  await banner.hover();
+  await footer.hover();
   await page.mouse.down();
-  divBox &&
-    (await page.mouse.move(divBox.x + divBox.width / 4, divBox.y, {
-      steps: 1000,
-    }));
+  businessInfoBox &&
+    (await page.mouse.move(
+      businessInfoBox.x,
+      businessInfoBox.y + businessInfoBox.height - 5,
+      {
+        steps: 1000,
+      }
+    ));
   await page.waitForTimeout(1000);
   await page.mouse.up();
   await expect(page).toHaveScreenshot();
```

Comment:
It might make sense to take screenshots for the component tree and preview instead of the whole page? I was thinking for here and on line 13

Reply:
In the second vs. third screenshots, the main difference is the state of the Save button, so we'd want to capture the whole screen to compare them properly. And then I thought we might as well capture the whole screen for the first screenshot too to make the comparison easier between the first and second screenshots

Reply:
Could we take a screenshot of just the save + deploy button? We could also just check that the two buttons both have the right enabled/disabled states. I think right now it's possible that the screenshot is large enough where even if the state of the buttons did change not enough pixels would change for the test to fail